installer/Languages/Default.isl

```diff
@@ -0,0 +1,384 @@
+﻿; *** Inno Setup version 6.1.0+ English messages ***

```

Comment:
I don't think we want our own Default.isl and Japanese.isl in the repository when the official files in the installation fit the bill.
Can you consider removing them?